### PR TITLE
fix(README): add some tips

### PR DESCRIPTION
**Pull request by WL2O2O**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,6 +13,22 @@ You are welcome to try out our dashboard directly with GreptimeDB's binary or wi
 - Visit `http://localhost:4000/dashboard`
 
 ### With Docker
+⭐We recommend using docker-compose to setup both greptimedb and dashboard.
+1. Use the `curl` command to get the `docker-compose` configuration file
+> Tips: If you are located in China and do not have access to an external network, you are likely to encounter network-related problems when executing the following commands.
```
Comment (ZonaHex) [inline README.md:18]:
```suggestion
```

File: README.md [hunk 2]
```diff
@@ -13,6 +13,22 @@ You are welcome to try out our dashboard directly with GreptimeDB's binary or wi
...
+```
+curl -O https://raw.githubusercontent.com/greptimeteam/dashboard/main/docker/docker-compose.yml
+```
+> Solution: Download the `docker-compose.yml` file locally and transfer it to the VM or server using `SFTP`. Or just copy the contents of [this file](https://github.com/GreptimeTeam/dashboard/blob/main/docker/docker-compose.yml) and create the file in the virtual machine or server.
```
Comment (ZonaHex) [inline README.md:22]:
```suggestion
> Tips: If you encounter network-related problems when executing the `curl` commands. Download the `docker-compose.yml` file manually or simply copy and paste the [contents](https://github.com/GreptimeTeam/dashboard/blob/main/docker/docker-compose.yml) into an empty `yml` file on your machine or server, based on what you need.
```

**Review by ZonaHex — COMMENTED** [review]
Hi, thanks a lot for your PR! I made some suggestions. You are more than welcome to share your thoughts furthermore.

File: README.md (hunk 2 above)
Comment (WL2O2O) [reply README.md:22]: You're welcome, thanks for the reply! I think this is a great suggestion on your part, and much more clearly articulated than its predecessor!👍🤝

File: README.md [hunk 3]
```diff
@@ -13,6 +13,21 @@ You are welcome to try out our dashboard directly with GreptimeDB's binary or wi
 - Visit `http://localhost:4000/dashboard`
 
 ### With Docker
+⭐We recommend using docker-compose to setup both greptimedb and dashboard.
+1. Use the `curl` command to get the `docker-compose` configuration file
+```
+curl -O https://raw.githubusercontent.com/greptimeteam/dashboard/main/docker/docker-compose.yml
+```
+> Tips: If you encounter network-related problems when executing the `curl` commands. Download the `docker-compose.yml` file manually or simply copy and paste the [contents](https://github.com/GreptimeTeam/dashboard/blob/main/docker/docker-compose.yml) into an empty `yml` file on your machine or server, based on what you need.
```
Comment (ZonaHex) [inline README.md:21]:
```suggestion
```
> Tips: If you encounter network-related problems, you could download the `docker-compose.yml` file manually or simply copy and paste the [contents](https://github.com/GreptimeTeam/dashboard/blob/main/docker/docker-compose.yml) into an empty `yml` file on your machine or server, based on what you need.
```

Comment (ZonaHex) [inline README.md:21]:
```suggestion
> Tips: If you encounter network-related problems, you can download the `docker-compose.yml` file manually or simply copy and paste the [contents](https://github.com/GreptimeTeam/dashboard/blob/main/docker/docker-compose.yml) into an empty `yml` file on your machine or server, based on what you need.
```

**Review by ZonaHex — APPROVED** [review]
Hi! Thanks again for your work. I made another suggestion and after this I think we are good to go!

**Review by WL2O2O — COMMENTED** [review]
nice

**Comment by WL2O2O** [comment]
Tkanks!!! Maybe this PR is insignificant, but it might give me more confidence to embrace open source!

**Comment by WL2O2O** [comment]
Tkanks!!! Maybe this PR is insignificant, but it might give me more confidence to embrace open source! @ZonaHex

**Comment by ZonaHex** [comment]
> Tkanks!!! Maybe this PR is insignificant, but it might give me more confidence to embrace open source! @ZonaHex

You are welcome! Every PR is valued and appreciated. This is what open source is all about :)

**Pull request merged**